software/genmap/index/main.nf

```diff
@@ -0,0 +1,38 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName } from './functions'
+
+params.options = [:]
+options        = initOptions(params.options)
+
+process GENMAP_INDEX {
+    tag '$fasta'
+    label 'process_high'
+    publishDir "${params.outdir}",
+        mode: params.publish_dir_mode,
+        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), meta:[:], publish_by_meta:[]) }
```

Comment:
```suggestion
        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:'index', meta:[:], publish_by_meta:[]) }
```

Reply:
For consistency with other indexing modules e.g. [bwa index](https://github.com/nf-core/modules/blob/master/software/bwa/index/main.nf)